pkg/controllers/deprovisioning/events/events.go

```diff
@@ -93,7 +93,7 @@ func Terminating(node *v1.Node, nodeClaim *v1beta1.NodeClaim, reason string) []e
 			InvolvedObject: node,
 			Type:           v1.EventTypeNormal,
 			Reason:         "DeprovisioningTerminating",
-			Message:        fmt.Sprintf("Deprovisioning Node: %s", cases.Title(language.Und, cases.NoLower).String(reason)),
+			Message:        fmt.Sprintf("Disruption Node: %s", cases.Title(language.Und, cases.NoLower).String(reason)),
```

Comment:
```suggestion
			Message:        fmt.Sprintf("Disrupting Node: %s", cases.Title(language.Und, cases.NoLower).String(reason)),
```

Reply:
Whoops, looks like the refactor monster got me again.